src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -34,7 +34,7 @@ object Boot extends App {
     // we need an ActorSystem to host our application in
     implicit val system = ActorSystem("rawls")
 
-    val slickDataSource = DataSource(DatabaseConfig.forConfig[JdbcProfile]("slick", conf))
+    val slickDataSource = DataSource(DatabaseConfig.forConfig[JdbcDriver]("slick", conf))
```

Comment:
switched to use JdbcDriver which is further down on the class hierarchy than JdbcProfile so we can access quoteIdentifier so our tests will work with H2 and MySql